### PR TITLE
initial pass at structuring docs, more to come

### PR DESCRIPTION
**Pull request by christian-posta**
Fixes https://github.com/solo-io/service-mesh-hub/issues/595

### CODE REVIEW

**Comment by Krief** [comment]
http://localhost:1313/reference/ -- This page has broken links

File: docs/content/getting_started/_index.md [hunk 1]
```diff
@@ -2,60 +2,106 @@
...
-We will assume in this guide that we have the following two contexts available in our kubeconfig file.
-Your actual context names will likely be different.
+## Getting `meshctl`
 
-* management-plane-context
-    - kubeconfig context pointing to a cluster where we will install and operate Service Mesh Hub
-* remote-cluster-context
-    - kubeconfig context pointing to a cluster where we will install and manage a service mesh using Service Mesh Hub 
-    
+Service Mesh Hub has a CLI tool called `meshctl` that helps bootstrap Service Mesh Hub, register clusters, install meshes, and more. Get the latest `meshctl` from the [releases page on solo-io/service-mesh-hub](https://github.com/solo-io/service-mesh-hub/releases).
```
Comment (EItanya) [inline docs/content/getting_started/_index.md:12]: Can we also include the following as an option:
`curl -sL https://run.solo.io/meshctl/install | sh`

**Comment by rvennam** [comment]
Multicluster communication task now a 404 from this task:
https://docs.solo.io/service-mesh-hub/latest/getting_started/guides/access_control_intro/

**Comment by rvennam** [comment]
Looks like it was just bad cache

**Pull request merged**